cim-client-sdk/src/main/java/com/crossoverjie/cim/client/sdk/ClientBuilder.java

```diff
@@ -20,4 +22,5 @@ public interface ClientBuilder {
     ClientBuilder okHttpClient(OkHttpClient okHttpClient);
     ClientBuilder messageListener(MessageListener messageListener);
     ClientBuilder callbackThreadPool(ThreadPoolExecutor callbackThreadPool);
+    ClientBuilder backoffStrategy(String backoffStrategy);
```

Comment:
```suggestion
    ClientBuilder backoffStrategy(BackoffStrategy backoffStrategy);
```

It is recommended to use the interface directly so that reflection is not needed.